onepassword/provider.go

```diff
@@ -35,7 +35,7 @@ func init() {
 	// }
 }
 
-// Provider The 1Password Connect terraform provider
+// Provider The 1Password Terraform Provider
```

Comment:
```suggestion
// Provider The 1Password Terraform provider
```

Just for consistency if we go with keeping "provider" lowercase.